sys/decl.go

```diff
@@ -52,6 +52,13 @@ func (t TypeCommon) Default() uintptr {
 	return 0
 }
 
+type Endianness int
+
+const (
+	LittleEndian Endianness = iota
```

Comment:
Please let's do just 'BigEndian bool'.
This `Endianness Endianness` causes too much clutter for no apparent benefit. There are only 2 options, which is naturally represented by a bool. There are multiple copies 

```
	Endianness: LittleEndian,

	Endianness == LittleEndian

	endianness := "LittleEndian"
	if !littleEndian {
		endianness = "BigEndian"
	}
```

If we do the bool, we can drop:
```
	endianness := "LittleEndian"
	if !littleEndian {
		endianness = "BigEndian"
	}
```
entirely, as a bool is correctly printed with %v.

We can also drop `Endianness: LittleEndian,` initialization, as default value for bool represents our semantic default of having little endian (unless said otherwise).